build.sh

```diff
@@ -347,19 +347,20 @@ isMSBuildOnNETCoreSupported()
         return
     fi
 
+    if [[ ("$__HostOS" == "Linux") && ("$__HostArch" == "x64" || "$__HostArch" == "arm64") ]]; then
+         __isMSBuildOnNETCoreSupported=1
+         # note: the RIDs below can use globbing patterns
```

Comment:
This is outdated, while you are here please just remove the rest of the unsupported rid logic.

Reply:
Done! I recall adding `-skipmanaged` to this script to eventually remove this hardcoded list. Great to have it finally happen.